Heleonix.Reflection/Reflector.cs

```diff
@@ -975,5 +978,35 @@ private static bool GetNextContainer(MemberInfo[] memberInfos, ref object contai
 
             return false;
         }
+
+        private static object CoerceValue(Type memberType, object value)
+        {
+            if (value is null)
+            {
+                throw new ArgumentNullException(nameof(value));
+            }
+
+            if (memberType.IsInstanceOfType(value))
```

Comment:
IsInstanceOfType is not available in netstandard1.6, so I get failed build on my machine.

Reply:
ah, oversight, should be `memberType.GetTypeInfo().IsInstanceOfType(value)`